.gitignore

```diff
@@ -3,7 +3,10 @@
 .output
 dist
 
-#package management and yarn3
+# Environment files
+.env*
```

Comment:
let's chat through this one. personally, I would recommend checking them in. Realistically your env file contains settings for each environment and then your secrets like passwords are added through runtime variables. (netlify variables for example)

Reply:
Okay, I'll remove it for now, because I really want to get these changes merged.